app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -583,14 +583,16 @@ def alert_valid_record?(alert)
       unless value_greater_than && is_integer?(value_greater_than)
         add_flash(_(template_error % [">", non, an_integer]), :error)
       end
-      unless value_greater_than.to_i.between?(0, 100)
```

Comment:
Not exactly, the thing is  a couple of lines below, on this validation:
` if value_less_than && value_greater_than && (value_less_than >= value_greater_than)` (Line 596)
 
We were comparing two strings, not two integers, so for example, the string "30" is less than "5" because is is a  lexicographical comparison.

```
2.3.0 :001 > "30" < "5"
 => true 
```

```
2.3.0 :002 > 30 < 5
 => false 
```